README.md

```diff
@@ -58,103 +61,151 @@ then accepted by a browser if the set conforms to its policy.
+-   Analytics/measurement of user journeys across O&O properties to improve quality of services.
+
 # Applications
 
-In support of the various browser privacy models, web platform features can use first-party sets to
-determine whether embedded content may or may not access its own state. For instance,
-sites may annotate individual cookies to be sent across same-party, cross-domain contexts by using
-the proposed [`SameParty` cookie attribute](https://github.com/cfredric/sameparty). It may also be reasonable to 
-use first-party sets to partition network caches, in cases where the tighter origin-based isolation
-is too expensive.
+In support of the various browser privacy models, first-party sets only control when embedded content that would otherwise be considered third-party can access its own state. Examples:
+
+-   Sites may annotate individual cookies to be sent across same-party, cross-domain contexts by using the proposed [SameParty cookie attribute](https://github.com/cfredric/sameparty).
+-   An iframe’s access to its own storage should be allowed when embedded on a site within the same first-party set. For example, this would allow a `https://b.example` iframe within `https://a.example` to access its own `https://b.example` databases, which would otherwise be restricted.
+
+Additionally, browsers may consider using First-Party Sets in new privacy features, such as:
+
+-   Top-level key for [partitioned cookies a.k.a “chips”](https://github.com/DCtheTall/CHIPS#partition-by-top-level-context). This allows third-party sites (such as embedded SaaS providers) to provide access to the same user session across multiple top-level sites within the same first-party set ([reference use-case](https://github.com/privacycg/first-party-sets/issues/33))
+-   Issuing WebID [directed identifiers](https://github.com/WICG/WebID/blob/main/directed_identifiers.md) by First-Party Set, so the same account can be shared across multiple applications or services provided by the same first-party.
+-   Applying [Privacy Budget](https://github.com/bslassey/privacy-budget) across an entire First-Party Set, in order to prevent fingerprinting entropy from being accumulated across domains that are able to communicate in an unconstrained manner due to access to cross-domain, same-party cookies.
+-   Top and/or second level key for cache partitioning, potentially with site opt-in.
+
+This proposal is consistent with the same-origin policy. That is, Web Platform features must not use first-party sets to make one origin's state directly accessible to another origin in the set. For example, if a.example and b.example are in the same first-party set, the same-origin policy would still prevent `https://a.example` from accessing `https://b.example`'s cookies or IndexedDB databases.
```

Comment:
This is confusing to me – however I think what is being left out is that there is a new class of cookies being considered - same party cookies - that **would** be able to be shared between a.example and b.example. Correct? If so, this should be called out here.

Reply:
Even with the `SameParty` cookie attribute; cookies bound to a.example can never be accessed by b.example. The intention for that attribute is that b.example can access **it's own cookies** when embedded on a.example, assuming that a.example and b.example are within the same FPS. Developers can then opt-in to `SameParty` attributes even on browsers that don't yet block third-party cookies by default.

Reply:
It still feels t me like this needs to be better spelled out in the PR - and if you're going to talk about cookies here then it feels like you should also mention the SameParty cookie proposal just to clarify how that enables or does not enable crossing the privacy boundary.

Reply:
Thanks for the feedback, Dan! I'll iterate on this in a separate PR (created #47 to track).